Sources/MapboxMaps/Annotations/ViewAnnotationManager.swift

```diff
@@ -9,16 +10,40 @@ public enum ViewAnnotationManagerError: Error {
     case geometryFieldMissing
 }
 
+/// An interface you use to detect when the map view lays out or updates visibility of annotation views.
+///
+/// When visible portion of a map changes, e.g. responding to the user interaction, the map view adjusts the positions and visibility of its annotation views.
+/// Implement methods of ``ViewAnnotationUpdateObserver`` to detect when the map view updates position/size for supplied annotation views.
+/// As well as when annotation views get show/hidden when going in/out of visible portion of the map.
+///
+/// To register an observer for view annotation updates, call the ``ViewAnnotationManager/addViewAnnotationUpdateObserver(_:)`` method.
+public protocol ViewAnnotationUpdateObserver: AnyObject {
+
+    /// Tells the observer that the frames of the annotation views changed.
+    ///
+    /// - Parameters:
+    ///   - annotationViews: The annotation views whose frames changed.
+    ///
+    func framesDidChange(for annotationViews: [UIView])
+
+    /// Tells the observer that the visibility of the annotation views changed.
+    ///
+    /// Use `isHidden` property to determine whether a view is visible or not.
+    /// - Parameters:
+    ///   - annotationsViews: The annotation vies whose visibility changed.
+    func visibilityDidChange(for annotationViews: [UIView])
+}
+
 /// Manager API to control View Annotations.
 ///
-/// View annotations are `UIView` instances that are drawn on top of the `MapView` and bound to some `Geometry` (only `Point` is supported for now).
+/// View annotations are `UIView` instances that are drawn on top of the ``MapView`` and bound to some ``Geometry`` (only ``Point`` is supported for now).
```

Comment:
Will these links for Geometry and Point do the right thing? Turf docs aren't well integrated at the moment.

Reply:
Not as such it seems. I did check that it generates links to those types, but there's nothing in the documentation pages for them. Doesn't look like a right thing, indeed. I'll revert these changes back.